tests/store/tracking/test_sqlalchemy_store_schema.py

```diff
@@ -21,7 +21,7 @@
     SqlModelVersionTag,
 )
 from mlflow.store.tracking.sqlalchemy_store import SqlAlchemyStore
-from tests.resources.db.initial_models import Base as InitialBase
+from mlflow.store.tracking.dbmodels.initial_models import Base as InitialBase
```

Comment:
Addressed [this TODO](https://github.com/mlflow/mlflow/pull/4180/files#diff-7e8f3a314f7ba34010ac4d0d42dd1e9714e10c6f7a82158b3c929bb7709521b4L1858)